controllers/complianceeventsapi/server.go

```diff
@@ -82,15 +84,38 @@ func init() {
 	ErrInvalidSortOption = fmt.Errorf(
 		"an invalid sort option was provided, choose from: %s", strings.Join(sortOptionsKeys, ", "),
 	)
+
+	// Reuse sortOptionsKeys since it's no longer being used. This overlaps with all the filterable columns.
+	validQueryArgs = sortOptionsKeys
```

Comment:
I don't get it this part. line 75 	
```
sortOptionsKeys = append(sortOptionsKeys, sortOptionsKeys)
			validQueryArgs = append(validQueryArgs, queryOption)
```
`sortOptionsKeys` append `sortOptionsKeys` same as  `validQueryArgs`

but now validQueryArgs is replaced by sortOptionsKeys?